src/Dialog/Dialog.js

```diff
@@ -398,7 +412,7 @@ class Dialog extends Component {
      */
     children: PropTypes.node,
     /**
-     * The css class name of the root element.
+     * The `className` to add to the root element.
```

Comment:
@akulov Sounds like we are not consistent on the master branch, at least we are on the next branch. Let's keep it like it then 👍 .